libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergEditorFragment.java

```diff
@@ -252,12 +253,20 @@ public void onCancelUploadForMediaDueToDeletedBlock(int mediaId) {
                         return otherMediaImageOptions;
                     }
 
+                    @Override
+                    public ArrayList<MediaOption> onGetOtherMediaFileOptions() {
+                        ArrayList<MediaOption> otherMediaDocumentOptions = initOtherMediaFileOptions();
+                        return otherMediaDocumentOptions;
+                    }
+
                     @Override
                     public void onOtherMediaButtonClicked(String mediaSource, boolean allowMultipleSelection) {
                         if (mediaSource.equals(MEDIA_SOURCE_STOCK_MEDIA)) {
                             mEditorFragmentListener.onAddStockMediaClicked(allowMultipleSelection);
                         } else if (mediaSource.equals(GIF_MEDIA)) {
                             mEditorFragmentListener.onAddGifClicked(allowMultipleSelection);
+                        } else if (mediaSource.equals(MEDIA_SOURCE_FILE)) {
+                            mEditorFragmentListener.onAddFileClicked(allowMultipleSelection);
                         }
```

Comment:
Could we convert this to a switch since another option was added?

Reply:
Done. Thanks.